src/mail/mail.service.ts

```diff
@@ -0,0 +1,35 @@
+import { MailerService } from '@nestjs-modules/mailer';
+import { Injectable } from '@nestjs/common';
+import { UserInvite } from './interface/user.interface';
+import { inviteConfig } from '../config/mail/invite/invite.config';
+import { configApp } from '../config/configApp';
+
+export const MAIL_SERVICE = 'MAIL SERVICE';
+
+export interface IMailService {
+  sendMail(user: UserInvite, token: string);
+}
+
+@Injectable()
+export class MailService implements IMailService {
+  constructor(private mailerService: MailerService) {}
+
+  async sendMail(user: UserInvite, token: string) {
```

Comment:
looks like the `sendMail` method is able to send only links for user invitations, right? I would make it more general, to be able to send mails with different payloads